### PR TITLE
commit_message quiz app

### PR DESCRIPTION
**Pull request by rezapanahi848**

### CODE REVIEW

File: javascript/QUIZ APP/javascript.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+      divAnswers.appendChild(button);
+      button.style.display = "block";
+      button.addEventListener("click", uponClicking);
+    }
+
+    // Creating button for randomizing the options
+    const buttonRandom = document.createElement("button");
+    buttonRandom.innerText = "Shuffle";
+    buttonRandom.setAttribute("class", "Shuffle");
```
Comment (islanderupnorth) [inline javascript/QUIZ APP/javascript.js:81]: Remember naming conventions and use `classList.add('shuffle')` which is made for handling class modifications

File: javascript/QUIZ APP/javascript.js
Comment (islanderupnorth) [inline javascript/QUIZ APP/javascript.js:1]: I believe you have placed the QUIZ APP in the incorrect folder?

File: javascript/QUIZ APP/style.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+form > div > div#answers > button:disabled {
+  cursor: not-allowed;
+}
+
+form > div > div#answers > button.Shuffle {
+  background-color: #007bff;
+  color: white;
+  border-radius: 20px;
+  margin-top: 10px;
+}
```
Comment (islanderupnorth) [inline javascript/QUIZ APP/style.css:63]: I would look into [media queries](https://www.w3schools.com/css/css3_mediaqueries.asp) which can help you style depending on the screen size viewing it.

I can imagine when the browser is smaller (more narrow) then the Shuffle buttons look as you wish but when in big screen are "squashed".

File: javascript/QUIZ APP/javascript.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  const buttons = event.target.parentElement.querySelectorAll(".btnoptions");
+  const buttonOptions = Array.from(buttons).map((button) => button.innerText);
+  for (let i = buttonOptions.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [buttonOptions[i], buttonOptions[j]] = [buttonOptions[j], buttonOptions[i]];
+  }
+  for (let i = 0; i < buttons.length; i++) {
+    buttons[i].innerText = buttonOptions[i];
+  }
+}
```
Comment (islanderupnorth) [inline javascript/QUIZ APP/javascript.js:128]: When you press shuffle a couple of times and then press on an answer it highlights the wrong index/colour.

File: javascript/QUIZ APP/javascript.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    ],
+    explanation: "Kroner is the currency of Denmark.",
+  },
+];
+
+// Creating form and styling the form
+const form = document.createElement("form");
+document.body.appendChild(form);
+form.style.alignItems = "center";
+form.style.margin = "2%";
```
Comment (islanderupnorth) [inline javascript/QUIZ APP/javascript.js:52]: Giving margin a % isn't the best practice. If you want to create a responsive design then I would recommend look at the [media queries](https://www.w3schools.com/css/css3_mediaqueries.asp).

**Review by islanderupnorth — COMMENTED** [review]
Good job - and styling wise it is looking good! If you enjoy playing around with the CSS and styling then I can recommend becoming familiar with media queries so you can make responsive designs!

I believe this is only the homework submission for week 1? Week 2 requires search functionality, submitting questions in a form & saving them in an array.

Great work though Milad!

File: javascript/QUIZ APP/javascript.js (hunk 3 above)
Comment (rezapanahi848) [reply javascript/QUIZ APP/javascript.js:128]: i tried so hard to fix it but it didn't fix what do you think i should do

**Pull request merged**